pkg/koordlet/statesinformer/config.go

```diff
@@ -18,26 +18,33 @@ package statesinformer
 
 import (
 	"flag"
+	"time"
 
 	corev1 "k8s.io/api/core/v1"
 )
 
 type Config struct {
 	KubeletPreferredAddressType string
-	KubeletSyncIntervalSeconds  int
-	KubeletSyncTimeoutSeconds   int
+	KubeletSyncInterval         time.Duration
+	KubeletSyncTimeout          time.Duration
+	InsecureKubeletTLS          bool
+	KubeletReadOnlyPort         uint
 }
 
 func NewDefaultConfig() *Config {
 	return &Config{
 		KubeletPreferredAddressType: string(corev1.NodeInternalIP),
-		KubeletSyncIntervalSeconds:  30,
-		KubeletSyncTimeoutSeconds:   3,
+		KubeletSyncInterval:         30 * time.Second,
+		KubeletSyncTimeout:          3 * time.Second,
+		InsecureKubeletTLS:          false,
+		KubeletReadOnlyPort:         10255,
 	}
 }
 
 func (c *Config) InitFlags(fs *flag.FlagSet) {
-	fs.StringVar(&c.KubeletPreferredAddressType, "KubeletPreferredAddressType", c.KubeletPreferredAddressType, "The node address types to use when determining which address to use to connect to a particular node.")
-	fs.IntVar(&c.KubeletSyncIntervalSeconds, "KubeletSyncIntervalSeconds", c.KubeletSyncIntervalSeconds, "The interval at which Koordlet will retain datas from Kubelet.")
-	fs.IntVar(&c.KubeletSyncTimeoutSeconds, "KubeletSyncTimeoutSeconds", c.KubeletSyncTimeoutSeconds, "The length of time to wait before giving up on a single request to Kubelet.")
+	fs.StringVar(&c.KubeletPreferredAddressType, "kubelet-preferred-address-type", c.KubeletPreferredAddressType, "The node address types to use when determining which address to use to connect to a particular node.")
+	fs.DurationVar(&c.KubeletSyncInterval, "kubelet-sync-interval", c.KubeletSyncInterval, "The interval at which Koordlet will retain data from Kubelet.")
```

Comment:
Please add an example if the flag is duration type. Otherwise, I'm concerned with the misuse like `--interval=30`.

Reply:
good point!